lib/logster/message.rb

```diff
@@ -80,7 +80,8 @@ def env=(env)
     end
 
     def self.hostname
-      @hostname ||= `hostname`.strip! rescue "<unknown>"
+      command = Logster.config.use_full_hostname ? `hostname -f` : `hostname`
+      @hostname ||= command.strip! rescue "<unknown>"
```

Comment:
Oh the test is failing because `@hostname` is cached in the `Message` class by the first test case that runs and when the one you've added is run it will not re-compute `@hostname` and will just return the cached value. I guess we will need to have 2 instance variables one when `use_full_hostname` is true and the other when it's not.

Reply:
Cool, that the test is indeed useful!

I added a reset for the `@hostname` variable to the test, before and after it runs. 

Good enough?

Reply:
Yep, looks good. Thanks!